components/Buttons/src/MDCButton.h

```diff
@@ -146,6 +146,15 @@
  */
 - (void)resetElevationForState:(UIControlState)state;
 
+/*
+ Indicates whether the button should automatically update its font when the device’s
+ UIContentSizeCategory is changed.
+ 
+ If set to YES, this button will base its text font on MDCFontTextStyleButton.
+ */
+@property (nonatomic, readwrite, setter=mdc_setAdjustsFontForContentSizeCategory:)
+    BOOL mdc_adjustsFontForContentSizeCategory UI_APPEARANCE_SELECTOR;
```

Comment:
I would caution against adding a method declared by other classes in UIKit to an existing UIKit class.  In this case setAdjustsFontForContentSizeCategory was adding to UILabel, UITextField and UITextView in iOS 10.  I don't know if the method will be added to UIButton in iOS 11, but the chance is non-zero.  Added explicit callout to iOS' method.